bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -332,6 +332,18 @@ data:
         eval $sentinel_info_command
     }
 
+    HEADLESS_SERVICE="{{ template "common.names.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}"
+    SERVPORT=$(get_port "$HOSTNAME" "SENTINEL")
+    REDISPORT=$(get_port "$HOSTNAME" "REDIS")
+
+    {{- if .Values.sentinel.externalMaster.enabled }}
+        REDIS_SERVICE={{ .Values.sentinel.externalMaster.host | quote }}
+        SENTINEL_SERVICE_PORT={{ .Values.sentinel.externalMaster.port | quote }}
+    {{- else }}
+        REDIS_SERVICE="{{ template "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}"
+        SENTINEL_SERVICE_PORT=$(get_port "{{ include "common.names.fullname" . }}" "SENTINEL")
+    {{- end }}
```

Comment:
Same here, please fix the shell indentation after rendering.

Reply:
Done